src/ontology/foodon-edit.owl

```diff
@@ -58457,8 +58469,10 @@ SubClassOf(obo:FOODON_03480047 obo:FOODON_03480029)
 
 # Class: obo:FOODON_03490100 (food packaging)
 
-AnnotationAssertion(obo:IAO_0000114 obo:FOODON_03490100 obo:IAO_0000428)
-AnnotationAssertion(obo:IAO_0000115 obo:FOODON_03490100 "Type of container or wrapping defined by the main container material, the container form, and the material of the liner lids or ends. Also type of container or wrapping by form; prefer description by material first, then by form.  [FDA CFSAN 1995]"@en)
+AnnotationAssertion(obo:IAO_0000114 obo:FOODON_03490100 obo:IAO_0000122)
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_03490100 "A food contact material which encloses the food and can be in the type of a container, packaging or wrapping."@en)
+AnnotationAssertion(obo:IAO_0000116 obo:FOODON_03490100 "Existing definition was a legacy definition, not ontological and has been moved to comments."@en)
```

Comment:
When mentioning `existing` you are now referencing the current definition, not the retired one, as the context of it's application changed when the definition did. Recommend using "Previous" like you did in the heading of your comment.
```suggestion
AnnotationAssertion(obo:IAO_0000116 obo:FOODON_03490100 "Previous definition was a legacy definition, not ontological and has been moved to comments."@en)
```

Reply:
Resolved in later commits.